### PR TITLE
second qualification form

### PR DESCRIPTION
**Pull request by harel**
Note that this form is not linked to from anywhere, and is accessible only via a link in an email campaign.

The url is : 
https://www.great.gov.uk/international/invest/request-call/?emt_id=44

To do (delete all that do not apply):

 - [x] Changelog entry added.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [x] (if changing the UI) Add a printscreen to the PR

### CODE REVIEW

**Comment by harel** [comment]
Screenshot from 2020-09-28 23-53-13

File: second_qualification/forms.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        self.utm_data = utm_data
+        self.submission_url = submission_url
+        self.enquiry_id = emt_id
+        super().__init__(*args, **kwargs)
+
+    def get_context_data(self):
+        data = self.cleaned_data.copy()
+        return {
+            'form_data': (
+                # (_('Email address'), data['email']),
```
Comment (richtier) [inline second_qualification/forms.py:71]: if not needed then deleting would make more readabke

**Review by richtier — APPROVED** [review]

File: .coveragerc [hunk 2]
```diff
@@ -7,3 +7,4 @@ omit =
     manage.py
     **/tests/*
     *__init__*
+    **/apps.py
```
Comment (depsiatwal) [inline .coveragerc:10]: any reason why we added

**Review by agiamas — APPROVED** [review]
lgtm

File: .coveragerc (hunk 2 above)
Comment (harel) [reply .coveragerc:10]: this is a django generated file, which the coverage code kept complaining has no tests. there is no need to test an automatically generated boilerplate by django so added it to the omit list.

**Pull request merged**